ecs-init/volumes/efs_mount_helper.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package volumes
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+const (
+	// MountBinary is the binary name of EFS Mount
+	MountBinary = "mount"
+	// UnmountBinary is the binary name for EFS Unmount
+	UnmountBinary = "unmount"
+)
+
+// MountHelper contains fields and methods for mounting and unmounting EFS volumes
+type MountHelper struct {
+	MountType string
+	Device    string
+	Target    string
+	Options   string
+	NetNSPid  int
+}
+
+// Mount helps mount EFS volumes
+func (m *MountHelper) Mount() error {
+	args := []string{}
+	if m.MountType != "" {
+		args = append(args, "-t", m.MountType)
+	}
+	if m.Options != "" {
+		args = append(args, "-o", m.Options)
+	}
+	args = append(args, m.Device, m.Target)
+
+	if err := m.Validate(); err != nil {
+		return err
+	}
+	return RunMount(args)
+}
+
+var RunMount = RunMountCommand
+
+func RunMountCommand(args []string) error {
```

Comment:
these probably don't need to be exported

Reply:
ah yeah, used in the same package! will change